Public/Sdk/Public/Transformers/Transformer.ToolDefinition.dsc

```diff
@@ -44,6 +44,12 @@ namespace Transformer {
         /** True if the executable depends on Windows directories. This signals that accesses to the Windows Directories should be allowed. */
         dependsOnWindowsDirectories?: boolean;
 
+        /** 
+          * True if the executable depends on directories that comprise the current host OS. 
+          * For instance on windows this signals that accesses to the Windows Directories %WINDIR% should be allowed. 
+          */
+        dependsOnCurrentHostOSDirectories?: boolean;

```

Comment:
Why do you have this both here and in `Transformer.ExecuteArguments`?  Only here should be sufficient.

Reply:
See comment on other place. My hope is to create symetry between execute and tooldefinitoin where tooldefinition is merely a set of presets to use

---
In reply to: [274228401](https://github.com/Microsoft/BuildXL/pull/125#discussion_r274228401) [](ancestors = 274228401)